src/installing-workflow/configuring-postgres.md

```diff
@@ -0,0 +1,58 @@
+# Configuring Postgres
+
+Deis Workflow's controller component relies on a PostgreSQL database to store platform state.
+
+By default, Deis Workflow ships with the [database] component, which provides an in-cluster PostgreSQL database backed up to in-cluster or off-cluster [object storage]. Currently, for object storage, which is utilized by _several_ Workflow components, only off-cluster solutions such as S3 of GCS are recommended in production environments. Experience has shown that many operators already opting for off-cluster object storage similarly prefer to host Postgres off-cluster as well, using Amazon RDS or similar. When excercising both options, a Workflow installation becomes entirely stateless, and is thus restored or rebuilt with greater ease if the need should ever arise.
```

Comment:
nit: can you shorten this a tiny by saying "... thus restored or rebuilt with greater ease should the need ever arise."